### PR TITLE
Projects activate

### PR DESCRIPTION
**Pull request by gvdent**
Fixes the K8s project controller to be compatible to the SingleDatabaseController.

### CODE REVIEW

File: src/project/KubernetesProjectCtrl.coffee [hunk 1]
```diff
@@ -9,26 +10,38 @@ Promise     = require('bluebird')
...
-  Promise.reject()
+doCall = (suffix) ->
+  uri = createUri(suffix)
+  rp({
+    uri: uri
+    json: true
+  })
+
+bus.on('project', (res, req) ->
+  Promise.resolve(doCall("list"))
```
Comment (malamili) [inline src/project/KubernetesProjectCtrl.coffee]: already returning a promise..

File: src/project/KubernetesProjectCtrl.coffee [hunk 2]
```diff
@@ -9,26 +10,38 @@ Promise     = require('bluebird')
...
+
+expect('id').bus('project.ready').do((res, req, id) ->
+  doCall("status/#{id}").then((status) ->
+    { ready: status.ready }
+  )
+)
+
+bus.on('getDatabaseForProject', (id) ->
+  doCall("status/#{id}").then((status) ->
+    Promise.resolve(status.service)
```
Comment (malamili) [inline src/project/KubernetesProjectCtrl.coffee]: same here..

Comment (gvdent) [inline src/project/KubernetesProjectCtrl.coffee]: This one does process the result though

File: src/project/KubernetesProjectCtrl.coffee (hunk 1 above)
Comment (gvdent) [inline src/project/KubernetesProjectCtrl.coffee]: 👍

File: src/project/KubernetesProjectCtrl.coffee (hunk 2 above)
Comment (malamili) [inline src/project/KubernetesProjectCtrl.coffee]: The  then() returns a promise (which allows for that chaining that we de), so the last line is always resolved..

```
..then((service) -> 
  status.service
)
```

**Pull request merged**